leet_69.py

```diff
@@ -0,0 +1,7 @@
+import math
+class Solution:
+    def mySqrt() -> int:
+        x = int(input("Please enter an integer:"))
```

Comment:
You generally don't prompt for inputs in leetcode problems. It'll automatically take input. You've changed the default signature of the function here. In original problem, the function signature is `def mySqrt(self, x):`.